TabulateSmarterTestAdminPackage.Common/Tabulation/CsvWriter.cs

```diff
@@ -22,49 +20,58 @@ public CsvWriter(TextWriter Writer)
 
         public CsvWriter(Stream stream, Encoding encoding = null, bool leaveOpen = false)
         {
-            if (encoding == null) encoding = sUTF8_NoBom;
+            if (encoding == null)
+            {
+                encoding = sUTF8_NoBom;
+            }
             mWriter = new StreamWriter(stream, encoding, cBufSize, leaveOpen);
         }
 
-        public CsvWriter(String path, bool append, Encoding encoding = null)
+        public CsvWriter(string path, bool append, Encoding encoding = null)
         {
-            if (encoding == null) encoding = sUTF8_NoBom;
+            if (encoding == null)
+            {
+                encoding = sUTF8_NoBom;
+            }
             mWriter = new StreamWriter(path, append, encoding, cBufSize);
         }
 
+        public void Dispose()
+        {
+            Dispose(true);
+        }
+
         public void Write(string[] values)
         {
-            for (int i = 0; i < values.Length; ++i)
+            for (var i = 0; i < values.Length; ++i)
             {
-                string value = values[i];
+                var value = values[i];
                 if (value == null)
                 {
                     // Do nothing
-                }
-                else if (value.IndexOfAny(sCsvSpecialChars) >= 0)
+                } else if (value.IndexOfAny(sCsvSpecialChars) >= 0)
```

Comment:
it's a resharper thing. I'll modify my resharper profile to dump those on their own lines and re-run against the appropriate directories.